app/assets/javascripts/discourse/app/mixins/upload-debugging.js

```diff
@@ -19,44 +25,65 @@ export default Mixin.create({
   },
 
   _performanceApiSupport() {
-    performance.mark("testing support 1");
-    performance.mark("testing support 2");
-    const perfMeasure = performance.measure(
+    this._performanceMark("testing support 1");
+    this._performanceMark("testing support 2");
+    const perfMeasure = this._performanceMeasure(
       "performance api support",
       "testing support 1",
       "testing support 2"
     );
     return perfMeasure;
   },
 
+  _performanceMark(markName) {
```

Comment:
Performance Mark:

<img width="215" alt="Screenshot 2023-03-31 at 11 32 33 AM" src="https://user-images.githubusercontent.com/5259935/229016558-4e8456e0-43e2-430e-acec-7166e2115a9e.png">

Reply:
He'll smoke some meat, then smoke you on the racetrack